libs/coin-modules/coin-aptos/src/types/assets.ts

```diff
@@ -1,6 +1,8 @@
 import type { Asset } from "@ledgerhq/coin-framework/api/types";
 
-export type AptosAsset = Asset;
+export type AptosAsset = Asset<AptosTokenInformation>;
+
+export type AptosTokenInformation = { asset_type: string };
```

Comment:
we now have 3 different types to say the same, i would recommend

```ts
export type AptosToken = { // or AptosTokenInfo
  tokenType: string; // this is your asset_type
  contractAddress: string;
};
```

Reply:
done, thanks.